Adeptus Titanicus 2018.gst

```diff
@@ -16402,6 +16403,35 @@ result of a single D6.</description>
         <cost name=" Points" typeId="a731-e220-2d8a-41bf" value="0.0"/>
       </costs>
     </selectionEntry>
+    <selectionEntry id="a77a-601b-06c0-9ccc" name="=Interfector= Portents of Doom" publicationId="975a-00f4-pubN89746" page="26" hidden="false" collective="false" import="true" type="upgrade">
+      <rules>
+        <rule id="869c-9ea5-feb0-8426" name="=Interfector= Portents of Doom" publicationId="975a-00f4-pubN89746" page="26" hidden="false">
+          <description>Reveal this Stratagem during the Strategy phase. Any enemy Titan or Knight Banner within 12&quot; of a Legio Interfector Titan may not
+be issued Orders this round.</description>
+        </rule>
+      </rules>
+      <costs>
+        <cost name=" Stratagem Points" typeId="efbf-52f7-fd08-f329" value="0.0"/>
+        <cost name=" Points" typeId="a731-e220-2d8a-41bf" value="0.0"/>
+      </costs>
+    </selectionEntry>
+    <selectionEntry id="a926-7329-163c-d9a6" name="Dark Pledge" publicationId="975a-00f4-pubN87630" page="79" hidden="false" collective="false" import="true" type="upgrade">
```

Comment:
how come half of the  (existing) strats are units and half are upgrades?

Reply:
No good reason, probably just an ineffectual default left in place; I suppose they should all could be best considered upgrades? I'm not sure that they're really Units, Models, or Upgrades, but those are all the possible choices.